source/libs/utils.js

```diff
@@ -2,13 +2,42 @@ import {h} from 'dom-chef';
 import select from 'select-dom';
 import elementReady from 'element-ready';
 import domLoaded from 'dom-loaded';
+import OptionsSync from 'webext-options-sync';
+
+let options;
+const optionsPromise = new OptionsSync().getAll();
 
 /**
+ * Enable toggling each feature via options.
  * Prevent fn's errors from blocking the remaining tasks.
  * https://github.com/sindresorhus/refined-github/issues/678
- * The code looks weird but it's synchronous and fn is called without args.
  */
-export const safely = async fn => fn();
+export const enableFeature = async ({fn, id: _featureId = fn.name}) => {
```

Comment:
I really don't like this. I was initially +1 for this PR, but I think it's a bad idea to make every single feature an "option" that is "enabled", and therefore potentially "optional".